### PR TITLE
chore: Use py-rattler from conda-forge

### PR DESCRIPTION
**Pull request by pavelzw**
## Summary by CodeRabbit

* **Compatibility**
  * macOS Intel (`osx-64`) is no longer listed as a supported platform.

* **Dependency Updates**
  * Updated Textual to require version 8.2.8 or later within the 8.x series.
  * Updated py-rattler to require version 0.26.x.

### CODE REVIEW

File: pixi.lock
Comment (pavelzw) [inline pixi.lock:1]: # Dependencies

<details>
<summary>Explicit dependencies</summary>

|Dependency|Before|After|Change|Environments|
|-|-|-|-|-|
|hatchling|1.31.0||Removed|default on osx-64|
|lefthook|2.1.10||Removed|default on osx-64|
|mypy|2.3.0||Removed|*all envs* on osx-64|
|pixi-browse|0.0.0||Removed|*all envs* on osx-64|
|pre-commit-hooks|6.0.0||Removed|default on osx-64|
|prettier|3.9.3||Removed|default on osx-64|
|pytest|8.4.2||Removed|*all envs* on osx-64|
|pytest-cov|7.1.0||Removed|*all envs* on osx-64|
|pytest-xdist|3.8.0||Removed|*all envs* on osx-64|
|pytest_textual_snapshot|1.1.0||Removed|*all envs* on osx-64|
|python|3.14.6||Removed|{default, py314} on osx-64|
|python|3.13.14||Removed|py313 on osx-64|
|python-build|1.5.0||Removed|default on osx-64|
|ruff|0.16.0||Removed|default on osx-64|
|taplo|0.10.0||Removed|default on osx-64|
|textual-dev|1.8.0||Removed|*all envs* on osx-64|
|twine|6.2.0||Removed|default on osx-64|
|ty|0.0.63||Removed|*all envs* on osx-64|
|types-pyyaml|6.0.12.20260518||Removed|*all envs* on osx-64|
|typos|1.48.0||Removed|default on osx-64|
|vhs|0.11.0||Removed|*all envs* on osx-64|
|zizmor|1.28.0||Removed|default on osx-64|

</details>

<details>
<summary>Implicit dependencies</summary>

|Dependency|Before|After|Change|Environments|
|-|-|-|-|-|
|_python_abi3_support|1.0||Removed|*all envs* on osx-64|
|aiohappyeyeballs|2.7.1||Removed|*all envs* on osx-64|
|aiohttp|3.14.3||Removed|*all envs* on osx-64|
|aiohttp-jinja2|1.6||Removed|*all envs* on osx-64|
|aiosignal|1.4.0||Removed|*all envs* on osx-64|
|annotated-doc|0.0.4||Removed|*all envs* on osx-64|
|aom|3.14.1||Removed|*all envs* on osx-64|
|ast-serialize|0.6.0||Removed|*all envs* on osx-64|
|async-timeout|5.0.1||Removed|{default, py314} on osx-64|
|attrs|26.1.0||Removed|*all envs* on osx-64|
|backports|1.0||Removed|default on osx-64|
|backports.tarfile|1.2.0||Removed|default on osx-64|
|backports.zstd|1.6.0||Removed|default on osx-64|
|brotli-python|1.2.0||Removed|default on osx-64|
|bzip2|1.0.8||Removed|*all envs* on osx-64|
|c-ares|1.34.8||Removed|default on osx-64|
|ca-certificates|2026.7.22||Removed|*all envs* on osx-64|
|cairo|1.18.4||Removed|*all envs* on osx-64|
|certifi|2026.7.22||Removed|default on osx-64|
|charset-normalizer|3.4.9||Removed|default on osx-64|
|click|8.4.2||Removed|*all envs* on osx-64|
|colorama|0.4.6||Removed|*all envs* on osx-64|
|comrak|0.54.0||Removed|default on osx-64|
|coverage|7.15.2||Removed|*all envs* on osx-64|
|cpython|3.14.6||Removed|{default, py314} on osx-64|
|cpython|3.13.14||Removed|py313 on osx-64|
|dav1d|1.2.1||Removed|*all envs* on osx-64|
|dbus|1.16.2||Removed|*all envs* on osx-64|
|docutils|0.23||Removed|default on osx-64|
|editables|0.6||Removed|default on osx-64|
|exceptiongroup|1.3.1||Removed|*all envs* on osx-64|
|execnet|2.1.2||Removed|*all envs* on osx-64|
|ffmpeg|8.1.2||Removed|*all envs* on osx-64|
|font-ttf-dejavu-sans-mono|2.37||Removed|*all envs* on osx-64|
|font-ttf-inconsolata|3.000||Removed|*all envs* on osx-64|
|font-ttf-source-code-pro|2.038||Removed|*all envs* on osx-64|
|font-ttf-ubuntu|0.83||Removed|*all envs* on osx-64|
|fontconfig|2.18.1||Removed|*all envs* on osx-64|
|fonts-conda-ecosystem|1||Removed|*all envs* on osx-64|
|fonts-conda-forge|1||Removed|*all envs* on osx-64|
|freetype|2.14.3||Removed|*all envs* on osx-64|
|fribidi|1.0.16||Removed|*all envs* on osx-64|
|frozenlist|1.8.0||Removed|*all envs* on osx-64|
|gdk-pixbuf|2.44.7||Removed|*all envs* on osx-64|
|glslang|16.3.0||Removed|*all envs* on osx-64|
|gmp|6.3.0||Removed|*all envs* on osx-64|
|graphite2|1.3.15||Removed|*all envs* on osx-64|
|h2|4.4.0||Removed|default on osx-64|
|harfbuzz|14.2.1||Removed|*all envs* on osx-64|
|hpack|4.2.0||Removed|default on osx-64|
|hyperframe|6.1.0||Removed|default on osx-64|
|icu|78.3||Removed|*all envs* on osx-64|
|id|1.6.1||Removed|default on osx-64|
|idna|3.18||Removed|*all envs* on osx-64|
|importlib-metadata|9.0.0||Removed|default on osx-64|
|importlib_resources|7.1.0||Removed|default on osx-64|
|iniconfig|2.3.0||Removed|*all envs* on osx-64|
|jaraco.classes|3.4.0||Removed|default on osx-64|
|jaraco.context|6.1.2||Removed|default on osx-64|
|jaraco.functools|4.6.0||Removed|default on osx-64|
|jinja2|3.1.6||Removed|*all envs* on osx-64|
|json-c|0.18||Removed|*all envs* on osx-64|
|keyring|25.7.0||Removed|default on osx-64|
|lame|4.0||Removed|*all envs* on osx-64|
|lcms2|2.19.1||Removed|*all envs* on osx-64|
|lerc|4.1.0||Removed|*all envs* on osx-64|
|libabseil|20260526.0||Removed|*all envs* on osx-64|
|libass|0.17.5||Removed|*all envs* on osx-64|
|libbrotlicommon|1.2.0||Removed|*all envs* on osx-64|
|libbrotlidec|1.2.0||Removed|*all envs* on osx-64|
|libbrotlienc|1.2.0||Removed|*all envs* on osx-64|
|libcxx|22.1.8||Removed|*all envs* on osx-64|
|libdeflate|1.25||Removed|*all envs* on osx-64|
|libdovi|3.4.0||Removed|*all envs* on osx-64|
|libev|4.33||Removed|default on osx-64|
|libexpat|2.8.1||Removed|*all envs* on osx-64|
|libffi|3.5.2||Removed|*all envs* on osx-64|
|libfreetype|2.14.3||Removed|*all envs* on osx-64|
|libfreetype6|2.14.3||Removed|*all envs* on osx-64|
|libglib|2.88.2||Removed|*all envs* on osx-64|
|libharfbuzz|14.2.1||Removed|*all envs* on osx-64|
|libharfbuzz-devel|14.2.1||Removed|*all envs* on osx-64|
|libhwloc|2.13.0||Removed|*all envs* on osx-64|
|libhwy|1.4.0||Removed|*all envs* on osx-64|
|libiconv|1.18||Removed|*all envs* on osx-64|
|libintl|0.25.1||Removed|*all envs* on osx-64|
|libjpeg-turbo|3.2.0||Removed|*all envs* on osx-64|
|libjxl|0.12.0||Removed|*all envs* on osx-64|
|liblzma|5.8.3||Removed|*all envs* on osx-64|
|libmpdec|4.0.0||Removed|*all envs* on osx-64|
|libnghttp2|1.68.1||Removed|default on osx-64|
|libogg|1.3.5||Removed|*all envs* on osx-64|
|libopenvino|2026.2.1||Removed|*all envs* on osx-64|
|libopenvino-auto-batch-plugin|2026.2.1||Removed|*all envs* on osx-64|
|libopenvino-auto-plugin|2026.2.1||Removed|*all envs* on osx-64|
|libopenvino-hetero-plugin|2026.2.1||Removed|*all envs* on osx-64|
|libopenvino-intel-cpu-plugin|2026.2.1||Removed|*all envs* on osx-64|
|libopenvino-ir-frontend|2026.2.1||Removed|*all envs* on osx-64|
|libopenvino-onnx-frontend|2026.2.1||Removed|*all envs* on osx-64|
|libopenvino-paddle-frontend|2026.2.1||Removed|*all envs* on osx-64|
|libopenvino-pytorch-frontend|2026.2.1||Removed|*all envs* on osx-64|
|libopenvino-tensorflow-frontend|2026.2.1||Removed|*all envs* on osx-64|
|libopenvino-tensorflow-lite-frontend|2026.2.1||Removed|*all envs* on osx-64|
|libopus|1.6.1||Removed|*all envs* on osx-64|
|libplacebo|7.360.1||Removed|*all envs* on osx-64|
|libpng|1.6.58||Removed|*all envs* on osx-64|
|libprotobuf|7.35.1||Removed|*all envs* on osx-64|
|librsvg|2.62.3||Removed|*all envs* on osx-64|
|libsqlite|3.53.3||Removed|*all envs* on osx-64|
|libtiff|4.7.2||Removed|*all envs* on osx-64|
|libusb|1.0.29||Removed|*all envs* on osx-64|
|libuv|1.52.1||Removed|*all envs* on osx-64|
|libvorbis|1.3.7||Removed|*all envs* on osx-64|
|libvpx|1.15.2||Removed|*all envs* on osx-64|
|libvulkan-loader|1.4.350.1||Removed|*all envs* on osx-64|
|libwebp-base|1.6.0||Removed|*all envs* on osx-64|
|libwebsockets|4.3.5||Removed|*all envs* on osx-64|
|libxml2|2.15.3||Removed|*all envs* on osx-64|
|libxml2-16|2.15.3||Removed|*all envs* on osx-64|
|libzlib|1.3.2||Removed|*all envs* on osx-64|
|linkify-it-py|2.1.0||Removed|*all envs* on osx-64|
|markdown-it-py|4.2.0||Removed|*all envs* on osx-64|
|markupsafe|3.0.3||Removed|*all envs* on osx-64|
|mdit-py-plugins|0.6.1||Removed|*all envs* on osx-64|
|mdurl|0.1.2||Removed|*all envs* on osx-64|
|more-itertools|11.1.0||Removed|default on osx-64|
|mpg123|1.32.9||Removed|*all envs* on osx-64|
|msgpack-python|1.2.1||Removed|*all envs* on osx-64|
|multidict|6.7.1||Removed|*all envs* on osx-64|
|mypy_extensions|1.1.0||Removed|*all envs* on osx-64|
|ncurses|6.6||Removed|*all envs* on osx-64|
|nh3|0.3.6||Removed|default on osx-64|
|nodejs|24.18.0||Removed|default on osx-64|
|openh264|2.6.0||Removed|*all envs* on osx-64|
|openssl|3.6.3||Removed|*all envs* on osx-64|
|packaging|26.2||Removed|*all envs* on osx-64|
|pango|1.56.4||Removed|*all envs* on osx-64|
|pathspec|1.1.1||Removed|*all envs* on osx-64|
|pcre2|10.47||Removed|*all envs* on osx-64|
|pixman|0.46.4||Removed|*all envs* on osx-64|
|platformdirs|4.11.0||Removed|*all envs* on osx-64|
|pluggy|1.6.0||Removed|*all envs* on osx-64|
|propcache|0.5.2||Removed|*all envs* on osx-64|
|psutil|7.2.2||Removed|*all envs* on osx-64|
|pugixml|1.15||Removed|*all envs* on osx-64|
|py-rattler|0.25.0||Removed|*all envs* on osx-64|
|pygments|2.20.0||Removed|*all envs* on osx-64|
|pyproject_hooks|1.2.0||Removed|default on osx-64|
|pysocks|1.7.1||Removed|default on osx-64|
|python-gil|3.14.6||Removed|{default, py314} on osx-64|
|python-gil|3.13.14||Removed|py313 on osx-64|
|python-librt|0.13.0||Removed|*all envs* on osx-64|
|python_abi|3.14||Removed|{default, py314} on osx-64|
|python_abi|3.13||Removed|py313 on osx-64|
|pyyaml|6.0.3||Removed|*all envs* on osx-64|
|readline|8.3||Removed|*all envs* on osx-64|
|readme_renderer|45.0||Removed|default on osx-64|
|requests|2.34.2||Removed|default on osx-64|
|requests-toolbelt|1.0.0||Removed|default on osx-64|
|rfc3986|2.0.0||Removed|default on osx-64|
|rich|15.0.0||Removed|*all envs* on osx-64|
|ruamel.yaml|0.19.1||Removed|default on osx-64|
|ruamel.yaml.clib|0.2.15||Removed|default on osx-64|
|sdl2|2.32.56||Removed|*all envs* on osx-64|
|sdl3|3.4.12||Removed|*all envs* on osx-64|
|shaderc|2026.3||Removed|*all envs* on osx-64|
|shellingham|1.5.4||Removed|*all envs* on osx-64|
|snappy|1.2.2||Removed|*all envs* on osx-64|
|spirv-tools|2026.2||Removed|*all envs* on osx-64|
|svt-av1|4.2.0||Removed|*all envs* on osx-64|
|syrupy|4.8.0||Removed|*all envs* on osx-64|
|tbb|2023.0.0||Removed|*all envs* on osx-64|
|textual|8.2.8||Removed|*all envs* on osx-64|
|textual-diff-view|0.1.5||Removed|*all envs* on osx-64|
|textual-serve|1.1.3||Removed|*all envs* on osx-64|
|tk|8.6.13||Removed|*all envs* on osx-64|
|tomli|2.4.1||Removed|*all envs* on osx-64|
|trove-classifiers|2026.6.1.19||Removed|default on osx-64|
|ttyd|1.7.7||Removed|*all envs* on osx-64|
|typer|0.27.2||Removed|*all envs* on osx-64|
|typing-extensions|4.16.0||Removed|{default, py314} on osx-64|
|typing_extensions|4.16.0||Removed|*all envs* on osx-64|
|tzdata|2026c||Removed|*all envs* on osx-64|
|uc-micro-py|2.0.0||Removed|*all envs* on osx-64|
|urllib3|2.7.0||Removed|default on osx-64|
|x264|1!164.3095||Removed|*all envs* on osx-64|
|x265|3.5||Removed|*all envs* on osx-64|
|yaml|0.2.5||Removed|*all envs* on osx-64|
|yarl|1.24.5||Removed|*all envs* on osx-64|
|zipp|4.1.0||Removed|default on osx-64|
|zstd|1.5.7||Removed|*all envs* on osx-64|
|[py-rattler](https://prefix.dev/channels/conda-forge/packages/py-rattler)|0.25.0|0.26.0|Minor Upgrade|*all envs* on {linux-64, osx-arm64, win-64}|

</details>

[^1]: **Bold** means explicit dependency.
[^2]: Dependency got downgraded.
